regulations/static/regulations/css/less/main.less

```diff
@@ -41,6 +41,8 @@ Modules
 @import "module/error.less";
 @import "module/paragraph-markers.less";
 @import "module/comment.less";
+@import "module/comment-review.less";
+@import "module/comment-media-queries.less";
```

Comment:
I thought about that. But when it comes time to refactor, most of the modules could be put into folders as well so I figure it would happen then.